v2/oauth/introduction.mdx

```diff
@@ -0,0 +1,138 @@
+**OIDC** extends **OAuth 2.0** by defining additional standard scopes, endpoints, authentication mechanisms, and response types, making it easier to provide user authentication in a standardized way.
+
+It also introduces a new type of token called **ID Token**.
+This token gets issued by the [**Authorization Server**](/#authorization-server). 
+It contains user profile information such as the user’s identity, the token’s issuance time, and its expiration time.
+Unlike [**Access Tokens**](/#access-token), the **ID Token** is specifically used for authentication purposes (to verify the user's identity). 
+
+#### Scopes
+
+**Scopes** are permissions granted by the [**Resource Owner**](#resource-owner) to the [**Client**](#client).
+For example, access only to read user emails or full access to user data.
+
+#### Redirect URIs
+
+A specific URI to which the [**Authorization Server**](/#authorization-server) redirects the [**Resource Owner's**](/#resource-owner) user-agent (e.g. browser) after the [**Client**](/#client) has been granted or denied the authorization request.
+This URI ensures that the [**Authorization Code**](/#authorization-code) or [**Access Token**](/#access-token) is sent to the correct application endpoint.
+
+Redirect URIs are used only in some of the **OAuth 2.0** flows. We will discuss them in detail below.
+
+#### Authorization Code
+
+An **Authorization Code** is a short-lived code that the [**Authorization Server**](/#authorization-server) provides to the [**Client**](/#client) via the [**Redirect URI**](#redirect-uri) after authorization has been granted.
+This code then gets exchanged for an [**Access Token**](/#access-token). 
+The **Authorization Code** flow enhances security by keeping tokens out of the user-agent and letting the [**Client**](/#client) manage the backend communication with the [**Authorization Server**](/#authorization-server).
+
+### Authorization Flows
+
+The **OAuth 2.0** protocol defines several *flows* to accommodate different use cases.
+They are a set of steps an **OAuth Client** has to perform in order to obtain an access token.
+
+Our implementation supports the following flow types:
+
+- [Authorization Code Grant](https://oauth.net/2/grant-types/authorization-code/): This flow it is best suited for **web applications** where the [**Client**](/#client) can securely store a client secret. It consists of the following steps:
+  - Client redirects the Resource Owner to the Authorization Server.
+  - Resource Owner approves the request and is redirected back to the Client with an authorization code.
+  - Client exchanges the authorization code for an access token at the Authorization Server.
+  - Access token is used to access resources on the Resource Server.
+
+- [Implicit](https://oauth.net/2/grant-types/implicit/): This flow is best suited for **public clients** like single-page applications (SPAs) without a backend server. It consists of the following steps:
```

Comment:
```suggestion
- [Implicit](https://oauth.net/2/grant-types/implicit/): This flow is best suited for **public clients** like single-page applications (SPAs) or mobile apps without a backend server. It consists of the following steps:
```

Reply:
This flow is deprecated, so i dont think we support this. Please check with @porcellus

Reply:
We support this technically. We could disable it fully if you prefer, but OIDC does allow it.

Reply:
resolved